lib/Date/Utility.pm

```diff
@@ -1167,15 +1167,17 @@ e.g.
 
 =cut
 
-sub _plus_years {
+sub plus_years {
     my ($self, $years) = @_;
```

Comment:
Do we need to change https://github.com/binary-com/perl-Date-Utility/pull/33/files#diff-f5a42ba81e6bad2f023037d764e416a3a8344f43734f9671dd3afd2e2b7c5488R1165 
to print Date::Utility->new('2000-02-29')->plus_years(1)->date_yyyymmdd; to consistence with https://github.com/binary-com/perl-Date-Utility/pull/33/files#diff-f5a42ba81e6bad2f023037d764e416a3a8344f43734f9671dd3afd2e2b7c5488R1202?